src/index.js

```diff
@@ -420,6 +420,16 @@ class LocalstackPlugin {
       });
     }
 
+    const addNetworks = async (containerID) => {
+      let networks = [];
```

Comment:
nitpick: The `networks` variable doesn't seem to be used - could be removed.

Reply:
It's superfluous; I forgot to remove it when I was cleaning up the commit. Will remove it and handle the other additions.